src/App.js

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
 import { Provider } from 'react-redux';
 import { createStore, applyMiddleware } from 'redux';
-import ReduxThunk from 'redux-thunk';
+import thunk from 'redux-thunk';
 import logger from 'redux-logger';
 import { RootNavigator } from './navigators';
-import { rootReducer } from './reducers';
+import { rootReducer } from '.';
```

Comment:
./

Reply:
Eslint throws: [eslint] Useless path segments for "./", should be "." (import/no-useless-path-segments)

Reply:
Do you have this in .eslintrc

```
{
  "extends": "airbnb",
  "parser": "babel-eslint",
  "ecmaFeatures": {
    "classes": true
  },
  "env": {
    "jest": true
  },
  "rules": {
    "arrow-parens": ["error", "as-needed"],
    "comma-dangle": "off",
    "react/jsx-filename-extension": [
      "error",
      { "extensions": [".js", ".jsx"] }
    ],
    "curly": ["error", "all"],
    "global-require": "off",
    "no-console": ["error", { "allow": ["warn", "error"] }],
    "no-use-before-define": ["error", { "variables": false }],
    "consistent-return": "off",
    "function-paren-newline": "off",
    "import/no-extraneous-dependencies": "off",
    "import/extensions": "off",
    "import/no-named-as-default": "off",
    "import/no-unresolved": "off",
    "import/newline-after-import": "off",
    "import/prefer-default-export": "off",
    "object-curly-newline": "off",
    "jsx-a11y/href-no-hash": "off",
    "jsx-a11y/img-has-alt": "off",
    "react/require-default-props": "off",
    "react/no-string-refs": "off",
    "react/no-array-index-key": "off",
    "space-before-function-paren": 0,
    "react/forbid-prop-types": "off"
  }
}
```